src/modernizer/violations.xml

```diff
@@ -0,0 +1,769 @@
+<?xml version="1.0"?>
```

Comment:
Split this to commit which imports original violations.xml and one which modifies it.